pkg/executor/bacalhau/bacalhau.go

```diff
@@ -26,221 +39,238 @@ type BacalhauExecutorOptions struct {
 }
 
 type BacalhauExecutor struct {
-	Options        BacalhauExecutorOptions
-	bacalhauEnv    []string
+	Options     BacalhauExecutorOptions
+	bacalhauEnv []string
+
 	bacalhauClient BacalhauClient
 	ipfsClient     *ipfs.Client
 }
 
 func NewBacalhauExecutor(options BacalhauExecutorOptions, ipfsClient *ipfs.Client) (*BacalhauExecutor, error) {
-	client, err := newClient(options)
+
+	// apiHost := ""
+	// if options.ApiHost != "DO_NOT_SET" {
+	// 	apiHost = fmt.Sprintf("BACALHAU_API_HOST=%s", options.ApiHost)
+	// }
+	// fmt.Printf("OPTIONS: %s\n", options)
+	// bacalhauEnv := []string{
+	// 	apiHost,
+	// 	fmt.Sprintf("HOME=%s", os.Getenv("HOME")),
+	// }
+
+	// log.Debug().Msgf("bacalhauEnv: %s", bacalhauEnv)
+	apiHost := fmt.Sprintf("http://%s:%s", "localhost", "20000")
```

Comment:
We already have the code to provide bacalhau host and port ... why are we undoing that?